packages/open-next/src/build.ts

```diff
@@ -175,34 +169,14 @@ function printOpenNextVersion(options: BuildOptions) {
   logger.info(`OpenNext v${options.openNextVersion}`);
 }
 
-function initOutputDir(srcTempDir: string, options: BuildOptions) {
+function initOutputDir(options: BuildOptions) {
   // We need to get the build relative to the cwd to find the compiled config
   // This is needed for the case where the app is a single-version monorepo and the package.json is in the root of the monorepo
   // where the build is in the app directory, but the compiled config is in the root of the monorepo.
-  const openNextConfig = readFileSync(
-    path.join(srcTempDir, "open-next.config.mjs"),
-    "utf8",
-  );
-  let openNextConfigEdge: string | null = null;
-  if (fs.existsSync(path.join(srcTempDir, "open-next.config.edge.mjs"))) {
-    openNextConfigEdge = readFileSync(
-      path.join(srcTempDir, "open-next.config.edge.mjs"),
-      "utf8",
-    );
-  }
   fs.rmSync(options.outputDir, { recursive: true, force: true });
-  const destTempDir = options.tempDir;
-  fs.mkdirSync(destTempDir, { recursive: true });
-  fs.writeFileSync(
-    path.join(destTempDir, "open-next.config.mjs"),
-    openNextConfig,
-  );
-  if (openNextConfigEdge) {
-    fs.writeFileSync(
-      path.join(destTempDir, "open-next.config.edge.mjs"),
-      openNextConfigEdge,
-    );
-  }
+  const { buildDir } = options;
+  fs.mkdirSync(buildDir, { recursive: true });
+  fs.renameSync(options.tempBuildDir, buildDir);
```

Comment:
We cannot use renameSync here.
Just tested on my computer and this crashed with this error : 
`EXDEV: cross-device link not permitted, rename '/tmp/.open-next-tempaOh2Rc' -> '/mnt/ssd2/projects/next-app-dir/.open-next/.build'`
We probably need to copy

Reply:
Oh different devices...
We could create the temp build folder in the app folder instead of the OS temp folder and that should solve the problem.
Thanks for catching this. 
I'll update tomorrow.

Reply:
I have updated to use use `cpSync`.
I picked `cpSync` over creating a folder in the current directory not to leave a folder in the current directory when it's not a next app.